src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -185,6 +185,30 @@ impl LocationState {
                 // need to be applied to this subtree.
                 _ => false,
             };
+            if self.permission.is_disabled() {
+                // A foreign access to a `Disabled` tag will have almost no observable effect.
+                // It's a theorem that `Disabled` node have no protected initialized children,
+                // and so this foreign access will never trigger any protector.
+                // Further, the children will never be able to read or write again, since they
+                // have a `Disabled` parents. Even further, all children of `Disabled` are one
```

Comment:
The argument could end here, right? The permissions below don't matter since anyway no access is possible.